packages/wallet-sdk/src/features/clients/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { Chain } from 'viem';
+import { baseSepolia, optimismSepolia, sepolia } from 'viem/chains';
+
+/**
+ * TODO[jake]: ensure all supported chains are added here
+ */
+
+// Currently only testnets are supported
+export const SUPPORTED_CHAIN_MAP = {
+  [baseSepolia.id]: baseSepolia,
+  [sepolia.id]: sepolia,
+  [optimismSepolia.id]: optimismSepolia,
+} as Record<number, Chain>;
```

Comment:
we prob don't want to introduce any blocking logic into SDK, just in case in the future we need to add support to certain chain, we don't block older SDK

i wonder if this supported maps and the if supported check is needed, given the SCWChain[] is returned by SCW FE and should always be valid

Reply:
Yeah I was torn on this. Its a great point. We need to be able to load the clients based on the chains and viem has some strict types. We might be able to create the clients based on just the rpc url. We do want to limit to testnet only for our first release